docs/user/time_series.md

```diff
@@ -0,0 +1,40 @@
+### Time Series Plots
+
+`plot_time_series` simply plots out a time series of any scalar beam parameter in the loaded in FITACF or RAWACF file. These include things like.... Currently there is no functionalilty to plot parameters from MAP files. To do that, you would need to manually extract the information from those loaded dictionaries. See [`SDarnRead`](code/SDarnRead.md) for more info.

```

Comment:
"Thse include things like...." needs to be expanded or removed.

Reply:
@billetd

Reply:
Oops, left in by accident.